app/src/main/java/com/communistutopia/spacetrader/model/Universe.kt

```diff
@@ -0,0 +1,78 @@
+package com.communistutopia.spacetrader.model
+
+import kotlin.random.Random
+import kotlin.math.pow
+import kotlin.math.sqrt
+
+/**
+ * Singleton object that handles all the logic for generating universe and other events
+ */
+object Universe {
+
+    // Constants
+    private val X_SIZE: Int = 150
+    private val Y_SIZE: Int = 100
+    private val NUM_SYSTEMS: Int = 10
+
+    private val solarSystems: ArrayList<SolarSystem> = ArrayList()
```

Comment:
I think Dr. Waters spoke in class about how we should program to abstractions, like List<SolarSystem>sys = new ArrayList<>() (in however Kotlin does it)